setup.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import print_function
+import sys
+if sys.version_info < (3, ):
+    print("Please use Python 3 for nnf python library")
+    sys.exit(-1)
+
+import platform
+python_min_version = (3, 6, 2)
+python_min_version_str = '.'.join(map(str, python_min_version))
+if sys.version_info < python_min_version:
+    print("You are using Python {}. Python >={} is required.".format(
+        platform.python_version(), python_min_version_str))
+    sys.exit(-1)
+
+import setuptools
+
+with open("README.md", encoding="utf-8") as f:
+    long_description = f.read()
+
+with open("requirements.txt") as f:
+    install_requires = f.read().splitlines()
+
+with open("VERSION") as f:
+    version = f.readline().strip()
+
+setuptools.setup(
+    name="nnfusion",
+    version=version,
+    author="Microsoft Corporation",
+    author_email="nnfusion@microsoft.com",
```

Comment:
Shall we use the external team mail alias "nnfusion-team" here?

Reply:
Sure